modules/videoio/include/opencv2/videoio/videoio_c.h

```diff
@@ -488,6 +491,13 @@ enum
     CV_CAP_OPENNI_QVGA_60HZ    = 4
 };
 
+// Supported output modes of Aravis image generator
+enum
+{
+    CV_CAP_MODE_GRAY8 = 0,      // Y8 (default)
```

Comment:
instead of starting an enum with all pixel formats possible, you should rather re-use the FOURCC property and pick the according formats from: https://linuxtv.org/downloads/v4l-dvb-apis/yuv-formats.html